cdap-proto/src/main/java/co/cask/cdap/proto/profile/Profile.java

```diff
@@ -38,6 +39,7 @@
   private final EntityScope scope;
   private final ProfileStatus status;
   private final ProvisionerInfo provisioner;
+  private final long creationTsSeconds;
```

Comment:
I appreciate being descriptive in this variable name, but this name will be part of the REST API, so we should try to be consistent with our other endpoints. We have mostly been using one word names. For example, the run records use 'start' and 'end' for the start and end timestamps. How about 'created' here?

Reply:
Makes sense, will rename to 'created'